gluon/_compat.py

```diff
@@ -73,6 +73,10 @@ def to_native(obj, charset='utf8', errors='strict'):
             return obj
         return obj.encode(charset, errors)
 
+    def u(x):
+        "safe alternative for the python 2 `unicode()` function, which doesn't exist in python 3"
+        return unicode('x', 'utf-8')
```

Comment:
the `x` parameter is not used at all

Reply:
hmm fine by me as well. didn't know that one :)

Reply:
and whoops - i almost create a bug

Reply:
thats what we have peer reviews for - glad you catched it

Reply:
I suggest closing this PR and opening a new one that uses to_native without the `u()` method definition

Reply:
I followed up your advise. Thanks for the feedback